server/src/dbschema.js

```diff
@@ -2,6 +2,12 @@ const db = require("./db");
 const Keygrip = require('keygrip');
 
 const createSQL = `
+
+CREATE TABLE IF NOT EXISTS images (
+  id VARCHAR(200) PRIMARY KEY,
+  image BYTEA
```

Comment:
Let's add a new random id, imageId, so that the clip and shot ids don't leak to users other than the creator.